send-transaction-service/src/transaction_client.rs

```diff
@@ -205,18 +206,20 @@ impl TransactionClient for TpuClientNextClient {
         wire_transactions: Vec<Vec<u8>>,
         stats: &SendTransactionServiceStats,
     ) {
-        let res = self
-            .sender
-            .try_send(TransactionBatch::new(wire_transactions));
-        match res {
-            Ok(_) => {
-                stats.send_attempt_count.fetch_add(1, Ordering::Relaxed);
-            }
-            Err(_) => {
-                warn!("Failed to send transaction transaction, transaction chanel is full.");
-                stats.send_failure_count.fetch_add(1, Ordering::Relaxed);
+        let mut measure = Measure::start("send-us");
+        self.runtime_handle.spawn({
+            let sender = self.sender.clone();
+            async move {
+                let res = sender.send(TransactionBatch::new(wire_transactions)).await;
+                if res.is_err() {
+                    warn!("Failed to send transaction to channel: it is closed.");
+                }
             }
-        }
+        });
+
+        measure.stop();
```

Comment:
We don't increment error counter here because there is no way to do this without changing the type of stats to `Arc<SendTransactionServiceStats>`. This matches the ConnectionCache version behavior, it also doesn't increment this counter (although it pretends it does)